src/Rings.jl

```
@@ -241,3 +241,50 @@ function is_finite(R::NCRing)
   c == 1 && return true
   throw(NotImplementedError(:is_finite, R))
 end
+
+@doc raw"""
+    is_noetherian(R::T) where T<:Ring
+
+Check if the ring $R$ is Noetherian. The method returns `true` or raises an error, if the check has not been implemented yet for $R$.
+```jldoctest
+julia> R, x = polynomial_ring(ZZ, [:x]);
+
+julia> is_noetherian(R)
+true
+```
+"""
+function is_noetherian(R::T) where T<:Ring
+  if R isa Union{Integers, <:Field}
+    return true
+  else
+    return error("is_noetherian and krull_dim are currently not implemented for rings of type $(typeof(R))")
+  end
+end
```

Why not only define 
```julia
is_noetherian

is_noetherian(::Field) = true
is_noetherian(::Integers) = true
```
and let the rest throw a `MethodError` (which is better for reflection utilities than the generic error you throw here)
the top one is needed as an anchor to attach the docstring to

---

I did not know, how to resolve the docstring thing, but I also like that there is a more detailed error message